src/store/asana/worker.ts

```diff
@@ -14,42 +18,71 @@ export function startWorkers() {
 
 async function processActions() {
   const asanaStore = useAsanaStore();
-  await processAction(asanaStore.actions, asanaStore.errors);
-  const nextTimeout = asanaStore.actions.length > 0 ? 0 : 1000;
-  self.setTimeout(() => {
-    processActions();
-  }, nextTimeout);
+  const actions = asanaStore.actions;
+  const errors = asanaStore.errors;
+
+  // eslint-disable-next-line
+  while (true) {
+    await processAction(actions, errors);
+    await sleep(1000);
+  }
 }
 
-async function processAction(actions: Action[], errors: AsanaError[]): Promise<void> {
+async function processAction(actions: Action[], errors: WorkerError[]): Promise<void> {
   while (actions.length > 0) {
     const action = actions[0];
+    if (action.isProcessing) {
+      errors.push({
+        message: "worker error",
+        description: "please reload page"
```

Comment:
Can we please make this a bit more descriptive? Something in the description (I think) that makes it clear why we are raising this error